docs/core/bluetooth/api.md

```diff
@@ -105,6 +105,37 @@ def _unavailable_callback(info: bluetooth.BluetoothServiceInfoBleak) -> None:
 cancel = bluetooth.async_track_unavailable(hass, _unavailable_callback, "44:44:33:11:23:42", connectable=True)
 ```
 
+### Finding out the availability timeout
+
+There is no availability timeout for connectable devices - a device must be connectable by at least one `connectable` controller or unavailable.
+
+For non-connectable devices, availability is based on the time since the device's last known broadcast. This timeout is learned automatically based on the device's regular broadcasting pattern. You can find out this with the `bluetooth.async_get_learned_advertising_interval` API.
```

Comment:
```suggestion
Availability is based on the time since the device's last known broadcast. This timeout is learned automatically based on the device's regular broadcasting pattern. You can find out this with the `bluetooth.async_get_learned_advertising_interval` API.
```

Since we don't stay connected all the time we still rely on the advertisement being recent to know if a connectable device is available or not.